mousetrap.js

```diff
@@ -886,9 +902,9 @@
         };
 
         // start!
-        _addEvent(targetElement, 'keypress', _handleKeyEvent);
-        _addEvent(targetElement, 'keydown', _handleKeyEvent);
-        _addEvent(targetElement, 'keyup', _handleKeyEvent);
+        _addEvent(targetElement, 'keypress', self._handleKeyEvent);
+        _addEvent(targetElement, 'keydown', self._handleKeyEvent);
+        _addEvent(targetElement, 'keyup', self._handleKeyEvent);
```

Comment:
This is inside mousetrap constructor, I use`self._handleKeyEvent` so that I can remove it on destroy